res/values/cr_strings.xml

```diff
@@ -1142,6 +1142,8 @@
     <string name="crdroid_website_summary">Visit our official website</string>
     <string name="crdroid_translate_title">Translations</string>
     <string name="crdroid_translate_summary">Help with localisation of ROM in your language!</string>
+    <string name="crdroid_sponsor_title">ScopeHosts</string>
+    <string name="crdroid_sponsor_summary">crDroid infrastructure is provided by ScopeHosts</string>
```

Comment:
Should it be "Visit our website host sponsor"

Reply:
Think more would click as would be interested what that is
If they see "sponsor", will bail out I guess